README.md

```diff
@@ -363,7 +363,7 @@ __authorizers and authenticators have to fit together__.
 
 Ember Simple Auth comes with 2 authorizers:
 
-* [`OAuth2BearerAuthorizer`](http://ember-simple-auth.com/api/classes/OAuth2BearerAuhtorizer.html): an OAuth 2.0 authorizer that uses Bearer tokens
+* [`OAuth2BearerAuthorizer`](http://ember-simple-auth.com/api/classes/OAuth2BearerAuthorizer.html): an OAuth 2.0 authorizer that uses Bearer tokens
```

Comment:
Do you want me to revert the change to the README but leave the change to the class comment?